test/node/connection-inspect.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const test = require('tap');
+
+const jstp = require('../..');
+
+const app = require('../fixtures/application.js');
+const application = new jstp.Application(app.name, app.interfaces);
+
+let server;
+let client;
+
+test.beforeEach((done) => {
+  server = jstp.tcp.createServer(0, [application]);
+  server.listen(() => {
+    const port = server.address().port;
+    client = jstp.tcp.createClient({ host: 'localhost', port });
+    done();
+  });
+});
+
+test.afterEach((done) => {
+  client.disconnect();
+  server.close();
+  done();
+});
+
+test.test('must process inspect packets', (test) => {
+  const expectedInterfaces = Object.keys(app.interfaces);
+  let expectedTests = 1;
+  expectedInterfaces.forEach((iface) => {
+    expectedTests += 1;
```

Comment:
@belochub I though It'll be more obvious, seems It's not, also I'm strongly opposed to leaving a comment. It should be written in a more obvious way that needs no explanation. Maybe I'll stop on my initial idea: also proposed by you:
```
let expectedTests = 1 + expectedInterfaces.length;
expectedTests +=  expectedInterfaces.reduce((methodsCount, iface) => {
    methodsCount += Object.keys(app.interfaces[iface]).length;
  });
```
Maybe I should merge this into one expression.
@metarhia/jstp do you have any ideas concerning this issue?